oshi-core/src/main/java/oshi/hardware/CentralProcessor.java

```diff
@@ -36,38 +36,38 @@
      *
      * @return vendor string.
      */
-    String getVendor();
+    @Deprecated String getVendor();
 
     /**
      * Name, eg. Intel(R) Core(TM)2 Duo CPU T7300 @ 2.00GHz
      *
      * @return Processor name.
      */
-    String getName();
+    @Deprecated String getName();
```

Comment:
While technically correct, for style purposes put in a newline after the `@Deprecated`.  Also add a `@deprecated` annotation to the javadoc (note the lower case) with a comment `replaced by {@link ProcessorIdentifier#getName()}`